tasks/main-blivet.yml

```diff
@@ -94,6 +103,30 @@
     packages_only: true
   register: package_info
 
+- name: load mount facts
+  setup:
+    gather_subset: '!all,!min,mounts'
+  register: __storage_mounts_after_packages
+
+- name: detect mount alteration by 'get required packages'
+  block:
+    - name: show mounts before manage the pools and volumes
+      debug:
+        var: __storage_mounts_before_packages.ansible_facts.ansible_mounts
+
+    - name: show mounts after manage the pools and volumes
+      debug:
+        var: __storage_mounts_after_packages.ansible_facts.ansible_mounts
+
+    - name: fail if mounts changed
+      fail:
+        msg: "get required packages changed mounts. Changed status is
+      {{ package_info.changed }}"
+  when:
+    - __storage_mounts_before_packages.ansible_facts.ansible_mounts |
+      count !=
+      __storage_mounts_after_packages.ansible_facts.ansible_mounts | count
+
```

Comment:
@dwlehman shouldn't we make it a separate test rather than asserting every time the role is used? This commit was intended to quickly illustrate the problem, but maybe it is not ideal to keep it in the long term.

Reply:
Weird, I tried to leave that one out for that reason. Do you know which commit it came with? Maybe one of the merge commits?

Reply:
Should be this:
Merge remote-tracking branch 'pcahyna/mount-assert' into non-destructive
57dd5ccf5f547c460d11ba2ab9434130e2a4a1cb